<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>widget</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Ajouter un Pokemon</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Nom :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="tailleLabel">
        <property name="text">
         <string>Taille :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="tailleLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="poidsLabel">
        <property name="text">
         <string>Poids :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="poidsLineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="pvLabel">
        <property name="text">
         <string>PV :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="pcLabel">
        <property name="text">
         <string>PC :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSlider" name="pcSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="typeLabel">
        <property name="text">
         <string>Type :</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="typeComboBox">
        <item>
         <property name="text">
          <string>Feu</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Elecrique</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Eau</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Plante</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="pvSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editButton">
        <property name="text">
         <string>Sauvegarder</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tbPokemon"/>
    </item>
    <item>
     <widget class="QPushButton" name="endButton">
      <property name="text">
       <string>Termin√©</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>widget</receiver>
   <slot>addPokemon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeButton</sender>
   <signal>clicked()</signal>
   <receiver>widget</receiver>
   <slot>removePokemon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editButton</sender>
   <signal>clicked()</signal>
   <receiver>widget</receiver>
   <slot>saveDB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>648</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>794</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addPokemon()</slot>
  <slot>removePokemon()</slot>
  <slot>saveDB()</slot>
 </slots>
</ui>
